src/views/compute/wizard/TemplateIsoSelection.vue

```diff
@@ -104,7 +104,6 @@ export default {
       const key = this.inputDecorator.slice(0, -2)
       for (const filter of this.filterOpts) {
         if (items[filter.id] && items[filter.id][key] && items[filter.id][key].length > 0) {
-          this.filterType = filter.id
```

Comment:
@utchoang Can we keep this so that when a template is searched and not found in the current tab, it automatically moves to the tab in which it is found

Reply:
@davidjumani Because this paragraph causes the pagination to jump back to the feature tab. I'll find a new way to fix it.